### PR TITLE
Fix/279 core resolve codacy issues

### PR DESCRIPTION
**Pull request by keskinmbaha**
Fixed some Codacy issues such as function declaration in loop and using == instead of ===. Similarly, fixed a misplaced '10' in parseInt, i.e. change parseInt(variable), 10 to parseInt(variable, 10)

### CODE REVIEW

File: source/components/home-page/home-page.js [hunk 1]
```diff
@@ -136,6 +133,33 @@ class HomePage extends YummyRecipesComponent {
     }
   }
 
+  /**
+   * Gets a next button element and adds an eventListener to it
+   * @param {} element - Next Button
```
Comment (jbutera94) [inline source/components/home-page/home-page.js:138]: Should this be `{Object}`? Not sure what `{}` does to JSDocs for the type.

File: source/components/home-page/home-page.js [hunk 2]
```diff
@@ -136,6 +133,33 @@ class HomePage extends YummyRecipesComponent {
...
+   */
+  nextButtonEventListener(element, elementId) {
+    element.addEventListener("click", () => {
+      this.recipeScroll(false, elementId);
+    });
+  }
+
+  /**
+   * Gets a prev button element and adds an eventListener to it
+   * @param {} element - Prev Button
```
Comment (jbutera94) [inline source/components/home-page/home-page.js:149]: Same as above.

**Review by jbutera94 — APPROVED** [review]
Looks good, noticed a small thing about the JSDocs comments.

File: source/components/home-page/home-page.js (hunk 2 above)
Comment (keskinmbaha) [reply source/components/home-page/home-page.js:149]: Will add, also thinking to consolidate the function into one and have another parameter as a boolean.

**Pull request merged**